src/components/MenuGroup/index.tsx

```diff
@@ -10,21 +10,28 @@ import './MenuGroup.scss';
 interface ComponentProps extends RouteComponentProps {
   title: string;
   urlBase: string;
+  dataTestId?: string;
```

Comment:
Don't do this.

Reply:
I’ll need it for this PR

https://github.com/thenewboston-developers/Website/pull/1461

Reply:
I think you can test that, without a test id too. It'll not be pretty tho. You'll have to use methods like `getByRole()`, `queryByText()` etc.

@angle943 thoughts?

Reply:
so this is certainly not the recommended way in the `react-testing-library` methodology, but I've been breaking a lot of their rules too to be honest(i'm too used to using `enzyme`).

Because I've been doing this in other areas, I think for now we can allow it. I may revisit our tests as a whole in the future and refine our tests to fit more with `react-testing-libraries`'s philosophy.

But at least until we have an end-to-end testing set up (via Cypress), I think we can be relaxed with what RTL recommends

Reply:
I found it quite difficult to query (without testid) unless I use something querySelectorAll and iterate through those items for navigationData... 

Agreed it wouldn’t be pretty either but that’s an alternative if we want to be strict